salt/modules/mac_timezone.py

```diff
@@ -85,6 +85,8 @@ def set_date(date):
     :return: True if successful, False if not
     :rtype: bool
```

Comment:
This still doesn't return a bool on error because `execute_return_success` returns a string.